section4/exercises/classMethods.js

```diff
@@ -28,13 +28,27 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
   }
 
-  // ADD CODE
+changeProtein() {
+  this.protein = "carne asada";
+  }
+
+addTopping(toppings) {
+  this.toppings.push(toppings);
```

Comment:
Here is an example where you pass in toppings as a parameter in line 41, and push the toppings into your array. Try to use this pattern for some of your other methods where you want to pass in information stored in a variable.